src/style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.calculator {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 40px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 50%;
+  width: 50%;
+  background-color: #f5f5f5;
+}
+
+.text-area {
+  width: 100%;
+  height: 100px;
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
+  align-items: center;
+  background-color: #939393;
+  color: #fff;
+  padding-inline-end: 10px;
+  font-size: 30px;
+}
+
+.button-area {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: rgba(169, 167, 167, 0.995);
+}
+
+.row {
+  width: 100%;
+  height: 100px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.row div {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid #cfd2cf;
+  font-size: 30px;
+  cursor: pointer;
+}
+
+.row .colored {
+  background-color: rgb(255, 128, 0);
+}
+
+.row .zero {
+  width: 200%;
+}
```

Comment:
- [ ] Good job so far with your design very impressive :clap: But note that is very important to make your project design color , size , information match with the the guiven design. Please sure that the `Calculator.js` component should match following given design :arrow_down: 

<img width="558" alt="calculator (1)" src="https://user-images.githubusercontent.com/6615394/188747737-8b85afd3-29d1-4539-bb96-1fbe01301e73.png">

### Your design
![Screenshot from 2022-09-07 19-29-23](https://user-images.githubusercontent.com/6615394/188962751-b3c6d8e6-5831-4d1b-83dc-588a5aa8b4a9.png)